shared/panels/contribution-sidepanel/_features/application-card/application-card.tsx

```diff
@@ -34,31 +31,32 @@ export function ApplicationCard({ application, contributionId, isIgnored }: Appl
     });
 
   function handleIgnoreApplication() {
-    ignoreApplicationMutate({ status: "IGNORED" });
+    ignoreApplicationMutate({ isIgnored: true });
   }
 
   function handleApproveApplication() {
     acceptApplicationMutate({
-      assignees: [applicant.githubUserId],
+      assignees: [contributor.githubUserId],
     });
   }
 
   return (
     <Paper background="transparent" border="none" classNames={{ base: "flex gap-md justify-between" }}>
       <div className="flex gap-lg">
-        <Avatar size="sm" shape="squared" src={applicant.avatarUrl} />
+        <Avatar size="sm" shape="squared" src={contributor.avatarUrl} />
 
         <div className="flex flex-col gap-md">
           <div className="flex flex-col">
             <Typo size="sm" weight="medium">
-              {applicant.login}
+              {contributor.login}
             </Typo>
 
-            <Typo size="xs" color="secondary">
-              {application.getApplicantTitle().wording} • {application.getRank()}
-              {" • "}
-              <Translate token={"panels:contributor.rank"} count={applicant.globalRankPercentile} />
-            </Typo>
+            {/* TODO https://linear.app/onlydust/issue/E-2154/[contributor]-add-global-ranking-info-to-contributorresponse */}
+            {/*<Typo size="xs" color="secondary">*/}
+            {/*  {application.getApplicantTitle().wording} • {application.getRank()}*/}
+            {/*  {" • "}*/}
+            {/*  <Translate token={"panels:contributor.rank"} count={applicant.globalRankPercentile} />*/}
+            {/*</Typo>*/}
```

Comment:
keep in mind for later, inside the applicant model create a new instance of RankModel